private-templates-service/client/src/components/AllCards/AllCards.tsx

```diff
@@ -22,10 +26,16 @@ import requireAuthentication from "../../utils/requireAuthentication";
 // Strings
 import { ALL_CARDS_LIST_VIEW, ALL_CARDS_GRID_VIEW, ALL_CARDS, ALL_CARDS_TITLE } from "../../assets/strings";
 import TagList from "./TagList";
+import { COLORS } from "../../globalStyles";
+
+
```

Comment:
Intended extra empty lines? Do you have a code policy checker on the pipeline for this kind of stuff?

Reply:
eslint should take care of it:)
removed